stubs/PyScreeze/pyscreeze/__init__.pyi

```diff
@@ -140,7 +140,7 @@ def locateCenterOnScreen(
 def locateCenterOnScreen(
     image: str | Image.Image,
     *,
-    minSearchTime: float,
+    minSearchTime: float = 0,
```

Comment:
You need to do the same thing at line 130 for the other `locateCenterOnScreen` overload.